modules/appeals_api/app/swagger/appeals_api/v1/nod_json_schema_swagger_helper.rb

```diff
@@ -53,19 +53,19 @@ def read_json_from_same_dir(filename)
   end
 
   def headers_json_schema
-    @headers_json_schema ||= read_json(['config', 'schemas', 'decision_reviews', 'v1', '10182_headers.json'])
+    @headers_json_schema ||= read_json(%w[config schemas decision_reviews v1 10182_headers.json])
```

Comment:
"Well, while I'm here I might as well sugar this up nice". 
![kevin-durant-mvp-gif-by-nba](https://github.com/department-of-veterans-affairs/vets-api/assets/8226466/0d170de6-fe27-4301-9498-13c326bca4c0)